src/style.css

```diff
@@ -0,0 +1,77 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(../src/fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(../src/fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+html {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+body {
+  --one-color: #687c94;
+  --two-color: #3d4e61;
+  --light-color: #e1e7ed;
+  --white--color: #fff;
+  --focus-color: #f6f6f7;
+  --input-color: #3d4e61;
+}
+
+.input {
+  display: flex;
+  width: 100%;
+  line-height: 70px;
+  border-radius: 4px;
+  border: 1px solid var(--light-color);
+  background: var(--white--color);
+  box-shadow: 0 1px 8px var(--input-color);
+  margin: 20px 0;
+  background-image: url(../src/images/Search.svg);
+  background-size: 19px;
+  background-repeat: no-repeat;
+  background-position: 26px;
+  padding: 0 62px;
+  font-size: 16px;
+}
+
+.input:hover {
+  /* box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2); */
```

Comment:
Remove comment. Try it write obvious code that explain all we need without comments)